BlazarUI/app/stylus/variables.styl

```diff
@@ -9,9 +9,11 @@ $font-normal = 400
 $font-medium = 500
 $font-heavy = 600
 
-$color-text-standard = #333
-$color-text-light = #555
-$color-text-lighter = #767676
+$color-text-standard = $color-heffalump
+$color-text-light = $color-eerie
+
+$link-color ?= $color-calypso
```

Comment:
the `?` isn't needed here (it's probably actually not really needed in the other place either)